### PR TITLE
Update current phase of Lisk SDK - Closes 6997

### LINKED ISSUE #6997: Update README
### Description

- Update beta phase description to properly describe current emerald, sapphire phase
- Fix badges

### Motivation

- Beta phase doesn't correctly mention our current phase description
- Build badge is no longer working

### PR DESCRIPTION
**Pull request by ishantiw**
### What was the problem?

This PR resolves #6997 

### How was it solved?

- Updated phase section of the readme

### How was it tested?

n/a

### CODE REVIEW

File: sdk/README.md [hunk 1]
```diff
@@ -12,15 +12,22 @@
...
 
-We strictly discourage anyone from using the beta release of the Lisk SDK for any production-based blockchain applications, i.e. a blockchain operating on a real mainnet. Throughout the beta phase, there will be significant changes in the Lisk protocol and implementation, which will eventually bring accessibility and reliability to a level that is feasible for production-based blockchain applications. At this time we only recommend the Lisk SDK for proof-of-concept blockchain applications, i.e. a blockchain operating on a fake testnet.
+We are currently working on [Sapphire phase](https://lisk.com/roadmap)[<img src="https://lisk.com/sites/default/files/2021-05/ico-sapphire.png" style="width:20px;vertical-align:middle">](https://lisk.com/roadmap) that brings the [Lisk interoperability solution blog](https://lisk.com/blog/research/lisk-interoperability-solution-published) which is fully researched and all Lisk Improvement Proposals (LIPs) are published and currently being actively developed. It consists of 8 roadmap objectives,
```
Comment (shuse2) [inline sdk/README.md:19]:
```suggestion
We are currently working on [Sapphire phase](https://lisk.com/roadmap)[<img src="https://lisk.com/sites/default/files/2021-05/ico-sapphire.png" style="width:20px;vertical-align:middle">](https://lisk.com/roadmap) that brings the [Lisk interoperability solution](https://lisk.com/blog/research/lisk-interoperability-solution-published) which is fully researched and all Lisk Improvement Proposals (LIPs) are published and currently being actively developed. It consists of 8 roadmap objectives,
```

File: sdk/README.md [hunk 2]
```diff
@@ -12,15 +12,22 @@
...
+- Define sidechain registration and lifecycle
+- Introduce token standards for the Lisk ecosystem
+- Introduce alternative validator selection mechanism for sidechains
+- Enhance signature scheme
+- Define state model and state root
+- Update Lisk-BFT for interoperability
+- Update block header format
 
-Please be advised, although we have stabilized the architecture of SDK, we cannot guarantee a blockchain application created with the beta release of the Lisk SDK will remain compatible with our planned release candidates.
+To learn more about the Lisk Interoperability and the current development status, check out [Lisk Improvement Proposal](https://github.com/LiskHQ/lips), [Research Forum](https://research.lisk.com/), [Lisk interoperability solution blog](https://lisk.com/blog/research/lisk-interoperability-solution-published) and [development board](https://github.com/orgs/LiskHQ/projects/10/views/4).
```
Comment (shuse2) [inline sdk/README.md:30]:
```suggestion
To learn more about the Lisk Interoperability and the current development status, check out [Lisk Improvement Proposal](https://github.com/LiskHQ/lips), [Research Forum](https://research.lisk.com/), [Lisk interoperability solution blog](https://lisk.com/blog/research/lisk-interoperability-solution-published) and [Lisk SDK v6.0.0 project board](https://github.com/orgs/LiskHQ/projects/10/views/4).
```

**Review by shuse2 — CHANGES_REQUESTED** [review]
Looks good to me, just minor suggestion

File: sdk/README.md (hunk 1 above)
Comment (mitsuaki-u) [inline sdk/README.md:19]:
```suggestion
We are currently working on [Sapphire phase](https://lisk.com/roadmap)[<img src="https://lisk.com/sites/default/files/2021-05/ico-sapphire.png" style="width:20px;vertical-align:middle">](https://lisk.com/roadmap) which is summarized in the [Lisk interoperability solution blog.](https://lisk.com/blog/research/lisk-interoperability-solution-published) All related Lisk Improvement Proposals (LIPs) are published and currently being actively developed. It consists of 8 roadmap objectives.
```

**Review by mitsuaki-u — CHANGES_REQUESTED** [review]
LGTM some minor changes

File: sdk/README.md (hunk 1 above)
Comment (ishantiw) [reply sdk/README.md]: sapphire phase is not summarised in Lisk interoperability solution blog but it is part of sapphire phase. Adding the rest of the suggested code in separate commit

**Review by shuse2 — APPROVED** [review]

File: sdk/README.md (hunk 1 above)
Comment (mitsuaki-u) [reply sdk/README.md]: Ok the sentence just reads strange. How does a phase "bring a blog", not sure if it is grammatically or semantically correct.

Comment (ishantiw) [reply sdk/README.md]: Like i said, the sentence is partially updated, please check the commit, there is no "blog"

Comment (mitsuaki-u) [reply sdk/README.md]: I see that you fixed it now. Good work!

Comment (mitsuaki-u) [reply sdk/README.md]: Actually it still reads strangely. Instead of the word "brings" I suggest "implements" or another word.

**Review by mitsuaki-u — APPROVED** [review]
One minor suggestion but approved from my end 👍

File: sdk/README.md (hunk 1 above)
Comment (ishantiw) [reply sdk/README.md]: since it is researched, designed and implemented, i thought it will be better to use "brings", I'll update it anyways

**Pull request merged**